openmdao/core/tests/test_check_totals.py

```diff
@@ -937,14 +937,14 @@ def setup(self):
                 obj = sub.add_subsystem('obj_cmp', om.ExecComp('obj = x**2 + z[1] + y1 + exp(-y2)', obj=0.0,
                                                          x=0.0, z=np.array([0.0, 0.0]), y1=0.0, y2=0.0),
                                   promotes=['obj', 'x', 'z', 'y1', 'y2'])
-                obj.declare_partials(of='*', wrt='*', method='cs')
+                obj.declare_partials(of='*', wrt='*', method='fd')
```

Comment:
why did this test switch from cs to fd?

Reply:
With cs it was failing with errors around 1e-9.  I just went back and took a look at this and it turns out that there was a bug on master in the code that switches over to FD to avoid nested CS.  A single variable name was being passed into `add_approximation` instead of a key tuple, so the name for the wrt variable inside of `add_approxmation` was wrong.  After fixing that bug on master, both branches now get identical total derivatives.  I updated the checking of the totals to use 2e-9 as the tolerance and now the test passes using cs.